src/main/services/staticSite.service.ts

```
@@ -0,0 +1,502 @@
+interface SqlBlock {
+  name: string;
+  sql: string;
+}
+
+function extractSqlBlocks(markdown: string): SqlBlock[] {
+  const blocks: SqlBlock[] = [];
+  // Match ```sql <name>\n...\n``` blocks
+  const regex = /```sql\s+(\w+)\s*\n([\s\S]*?)```/g;
+  let match: RegExpExecArray | null;
+  // eslint-disable-next-line no-cond-assign
+  while ((match = regex.exec(markdown)) !== null) {
```

Biome (2.5.0)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)